src/shared-with-pluot-core/script-message-channels/WebMessageChannel.js

```diff
@@ -48,7 +48,7 @@ export default class WebMessageChannel extends ScriptMessageChannel {
       if (!(evt.data && evt.data.what === IFRAME_MESSAGE_MARKER &&
         evt.data.action &&
         (!evt.data.from || evt.data.from === 'module') &&
-        (evt.data.callFrameId ? evt.data.callFrameId === callFrameId : true))) {
+        ((evt.data.callFrameId && callFrameId) ? evt.data.callFrameId === callFrameId : true))) {
```

Comment:
oh huh, was this resulting in...what? events getting sent to the wrong frame?

Reply:
This caused the version of daily-js used in the React demo (0.9.986) to hang on loading. I found this fairly late in testing. Frustratingly, I didn't manage to figure out how this bug crept in during the merges that happened over the course of last week. Possibly I messed up when fixing a merge conflict at some point. Ordinarily I would have just kept hunting until I found the exact change that broke things, but I really needed to push live streaming and this was the last blocker and after hunting for a while I ran out of brain cells. So I just fixed it here and tested ever version of daily-js against the React demo.

(Note that, confusingly, this change is a fix to pluot-core, even though it's in daily-js, because this file is part of the pluot-core build.)